specs/_features/maxeb_increase/capella.py

```diff
@@ -1316,7 +1359,7 @@ def get_unslashed_attesting_indices(state: BeaconState,
     output = set()  # type: Set[ValidatorIndex]
     for a in attestations:
         output = output.union(get_attesting_indices(state, a.data, a.aggregation_bits))
-    return set(filter(lambda index: not state.validators[index].slashed, output))
+    return set(filter(lambda index: not is_slashed_attester(state.validators[index]), output))
```

Comment:
I think we should have `is_slashed_validator(validator): return is_slashed_attester(validator) or is_slashed_proposer(validator)`, and use that here. Don't see why attestations from someone slashed as a proposer should count here

```suggestion
    return set(filter(lambda index: not is_slashed_validator(state.validators[index]), output))
```

Reply:
The intention of decoupling proposer and attester slashing states was to prevent proposer from suffering from attestation and inactivity leak penalties as well as the correlation penalty. So a proposer pays a fixed penalty for equivocation.

Both, `get_unslashed_attesting_indices` and `get_eligible_validator_indices` are used in computing those penalties hence the change in these functions. A way around that is to add `is_slashed_proposer` into specific places where penalties are calculated.

If we’re about to remove initial slashing penalty in general, maybe we can revert the decoupling slashings change to avoid unnecessary complexity.

Reply:
I see, didn’t realize they were used to calculate penalties as well. I think adding it into specific places would be preferable